### PR TITLE
Fit 372 sem ver for perry

### PR DESCRIPTION
**Pull request by AVandana-OSI**
### JIRA Issue Link
https://osi-cwds.atlassian.net/browse/FIT-372

### Technical Description
Testing the SemVer for Perry by updating the project version with the new version on Gradle file

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -70,6 +69,19 @@ project.ext {
...
+
+ if (System.getProperty('newVersion') && System.getProperty('newVersion')!='null') {
+    println "Using newer version"
+    def newVersion = "${System.getProperty('newVersion')}"
+    version = newVersion
+    projectMajorVersion = newVersion
+    projectVersion = newVersion
+  } else {
+    println "using older version"
+    version = projectMajorVersion
```
Comment (mariamcws) [inline build.gradle]: why not version = projectVersion
see line 102
and remove line 102

File: build.gradle [hunk 2]
```diff
@@ -70,6 +69,19 @@ project.ext {
...
+}
+
+def dockerTag = version
```
Comment (mariamcws) [inline build.gradle]: should it be def dockerTag = projectVersion

File: build.gradle [hunk 3]
```diff
@@ -298,4 +310,4 @@ javadoc {
 task integrationTest(type: Test, dependsOn: testClasses) {
     include '**/*IT*'
     exclude '**/*Test*'
-}
+}
```
Comment (mariamcws) [inline build.gradle]: no new line at eof

File: build.gradle (hunk 1 above)
Comment (AVandana-OSI) [reply build.gradle]: @mariamcws 102 line? in Project.exe they have defined as projectMajorVersion

**Pull request merged**